vpr/src/pack/pack.cpp

```diff
@@ -246,7 +246,7 @@ bool try_pack(t_packer_opts* packer_opts,
         }
 
         //Reset clustering for re-packing
-        g_vpr_ctx.mutable_clustering().clb_nlist = ClusteredNetlist();
+        //g_vpr_ctx.mutable_clustering().clb_nlist = ClusteredNetlist();
```

Comment:
Better to delete this code rather than commenting it out if it isn't needed anymore.